.changelog/25523.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+scheduler: log the items Slice instead of the idset to properly log this in JSON
```

Comment:
This code isn't executed in the scheduler, but on the client. Let's fix that and reword this to make it more sensible as a user-facing item, albeit a bit wordier:

```suggestion
client: Fixed a bug where JSON formatted logs would not show the requested and overlapping cores when failing to reserve cores
```